docs/NewOperators.md

```diff
@@ -0,0 +1,18 @@
+## How to implement a new operator in Glow
+
+> Example PR: https://github.com/pytorch/glow/pull/2334
+
+### Implementation
+* Create a new glow high level IR node in `ClassGen/NodeGen.cpp`.
+* Implement the `verify()` method for the new node in `Graph/Nodes.cpp`.
+* Implement a node creation method in `Graph/Graph.cpp`.
+* Implement logic to load models that contains the operator in `Importer/Caffe2ModelLoader.cpp` or `Importer/ONNXModelLoader.cpp` depending on which type of model the operator comes from. Add the operator to `Importer/CommonOperatorLoader.h` instead if the loading logic can be shared between Caffe2 and ONNX.
+* Create a new glow low level IR instruction in `ClassGen/InstrGen.cpp`
+* Implement the interpreter implementation of the operator in `Interpreter/InterpreterNodes.cpp`
```

Comment:
I'd also specify that there are several backends which needs to have operator implementation.
Interpreter, CPU, OpenCL. Having CPU implementation is highly encouraged as interpreter is basically a thing to play with but really hard to use for ONNXIFI and other scenarious.